src/main/groovy/edu/oregonstate/mist/osuevents/db/EventsDAOWrapper.groovy

```diff
@@ -14,11 +14,11 @@ class EventsDAOWrapper {
     }
 
     /**
-     * Get all events.
+     * Get all events within past time window, or all events if changedInPastHours is not passed
      * @return
      */
-    List<Event> getEvents() {
-        List<Event> events = eventsDAO.getEvents(null)
+    List<Event> getEvents(Integer changedInPastHours = null) {
```

Comment:
Defaults to null so that this method can be called without passing the changedInPastHours argument (if all events are desired)